tests/Query/BuilderTest.php

```diff
@@ -568,7 +568,7 @@ public function testInsertDatetime(): void
         $qb = $conn->table($tableName);
 
         $row = $this->generateTestRow();
-        $carbonMax = Carbon::maxValue();
+        $carbonMax = Carbon::create(9999, 12, 31, 23, 59, 59, 'UTC');
```

Comment:
memo: The maxValue and minValue functions were removed when Carbon v3 was introduced.
https://github.com/briannesbitt/Carbon/commit/a4726ae7633a4ae361a2561d2256900115ef2015

Laravel 11 now allows v3 series.
https://github.com/laravel/framework/blob/11.x/composer.json#L39